Task 9/script.js

```diff
@@ -0,0 +1,29 @@
+/* ------------------------------ TASK 9 ---------------------------------------------------------------
+Sukurkite konstruktoriaus funkciją "Movie" (naudokte ES6), kuri gebės sukurti objektus 3 savybėm ir 1 metodu.
+
+Savybės:
+title: string
+director: string
+budget: number
+
+Metodas: 
+wasExpensive() - jeigu filmo "budget" yra daugiau nei 100 000 000 mln USD, tada grąžins true, kitu atveju false. 
+------------------------------------------------------------------------------------------------------ */
+class Movie {
+    constructor(title, director, budget) {
+        this.title = title;
+        this.director = director;
+        this.budget = budget;
+    };
+    wasExpensive() {
+        if (this.budget > 100000000) {
+            return true;
+        } else {
+            return false;
+        }
```

Comment:
tokio tipo sąlygą reikėtų rašyti `return this.budget > 100000000`, kuri priklausomai nuo biudžeto ir grąžins reikšmę true arba false